htdocs/comm/propal/card.php

```diff
@@ -966,7 +966,7 @@
 				$tmpprodvat = price2num(preg_replace('/\s*\(.*\)/', '', $prod->tva_tx));
 
 				// if price ht is forced (ie: calculated by margin rate and cost price). TODO Why this ?
-				if (!empty($price_ht)) {
+				if (!empty($price_ht) || $price_ht === '0') {
```

Comment:
There is other case: 0.0 like when $price_ht is float.
In fact, for which value of $price_ht do you experience trouble ? (var_dump($price_ht))

Reply:
The value '0'  is equal to `false` and  `empty(0)` then is true